charts/kube-prometheus-stack/templates/prometheus-operator/clusterrole.yaml

```diff
@@ -19,16 +19,17 @@ rules:
   - monitoring.coreos.com
```

Comment:
Might be good to clean this up in line with https://github.com/prometheus-operator/prometheus-operator/blob/master/example/rbac/prometheus-operator/prometheus-operator-cluster-role.yaml

Reply:
I sync'd this up.

Reply:
I sync'd this up.